stdlib/asm/collections/smt.masm

```diff
@@ -370,7 +446,7 @@ end
 #! Cycles
 #!   Leaf empty: 48 cycles
 #!   Leaf single: 99 cycles
-#!   Leaf multiple: unimplemented
+#!   Leaf multiple: 184 cycles
```

Comment:
The cycle count is not fixed at 184, right? It should be variable based on the number of key-value pairs in the leaf.